.github/workflows/bump-and-release.yml

```diff
@@ -23,6 +23,7 @@ on:
           - Comscore
           - Conviva
           - SideloadedSubtitle
+          - Yospace
```

Comment:
will the publishing work, if we don't have access to their repo? (e.g. when `pod` does the linting in a new project, will it pass?)

Reply:
Yes, as we agreed we will not add the private yospace SDK as dependency.
Instead I covered it in the README doc how to retrieve the SDK.

Reply:
Yepp, I saw it, but what I mean is that the `[bump-and-release.yml workflow](https://github.com/THEOplayer/iOS-Connector/pull/49/files/e4923fa9ab78b43e6544292f5b4f71d7546ba4fe#diff-ad8c2f750a9d2cdab6881d96e37a234566f512aeab64185105593a254051592c)` will work for Yospace or we need to do the release (or something) manually to make it available on Cocoapods/SPM.

Reply:
It should pass because the podspec has valid properties and dependencies. The sandbox will try to fetch the pod with it's dependencies, but it does not try to compile the code. Thus no compiler error gets thrown ie. on:
`import YOAdManagement`
Dry-run also passed.